frontend/mission-components/components/experience/hooks/usePostProcessing.ts

```
@@ -0,0 +1,199 @@
+
+        // restore original resize method
+        if (originalResize) {
+          composer.setSize = originalResize as (width: number, height: number) => void;
+        }
+      } catch (error) {
+        console.error("Error cleaning up bloom effect:", error);
+      }
+    };
+  }, [composer]);
+
+  return composer;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)